falcon/response.py

```diff
@@ -185,23 +188,28 @@ def media(self, obj):
     def __repr__(self):
         return '<%s: %s>' % (self.__class__.__name__, self.status)
 
-    def set_stream(self, stream, stream_len):
-        """Convenience method for setting both `stream` and `stream_len`.
+    def set_stream(self, stream, content_length):
+        """Convenience method for setting both `stream` and `content_length`.
+
+        Args:
+            stream (str): stream
+            content_length (int): set the stream length, if known
```

Comment:
Nitpick: I think this could be improved a bit, e.g.: "Length of the stream to use for the Content-Length header in the response."